### PR TITLE
Corrected aliases error output and added IDs of the problem objects

### PR DESCRIPTION
**Pull request by GermanBluefox**
If target or source does not exist in aliases, function formatAliasValue crashed, but did not show the problem objects.
This PR corrects it.

### CODE REVIEW

File: packages/common/src/lib/common/tools.ts [hunk 1]
```diff
@@ -2609,6 +2613,12 @@ export function formatAliasValue(
     }
 
     if (targetObj?.alias?.read) {
+        if (!sourceObj) {
+            logger.error(
+                `${logNamespace} source in "${targetId}" not exist for "read" function: "${targetObj.alias.read}"`
```
Comment (Apollon77) [inline packages/common/src/lib/common/tools.ts:2618]:
```suggestion
                `${logNamespace}source in "${targetId}" does not exist for "read" function: "${targetObj.alias.read}"`
```

Comment (Apollon77) [reply packages/common/src/lib/common/tools.ts:2618]: And formally the "space too much after ${logNamespace} needs to be changed in whole function :-)

File: packages/common/src/lib/common/tools.ts [hunk 2]
```diff
@@ -2632,13 +2642,19 @@ export function formatAliasValue(
...
+                `${logNamespace} Invalid read function for "${targetId}": "${targetObj.alias.read}" => ${e.message}`
             );
             return null;
         }
     }
 
     if (sourceObj && sourceObj.alias && sourceObj.alias.write) {
+        if (!targetObj) {
+            logger.error(
+                `${logNamespace} target for "${sourceId}" not exist for "write" function: "${sourceObj.alias.write}"`
```
Comment (Apollon77) [inline packages/common/src/lib/common/tools.ts:2654]:
```suggestion
                `${logNamespace}target for "${sourceId}" does not exist for "write" function: "${sourceObj.alias.write}"`
```

**Review by Apollon77 — CHANGES_REQUESTED** [review]
Just some small stuff in texts and logging from my side

**Review by foxriver76 — APPROVED** [review]

**Pull request merged**